src/javascripts/ingredients/showIngredients.js

```diff
@@ -9,4 +9,16 @@ const showIngredients = (array) => {
   });
 };
 
-export default showIngredients;
+const showUserIngredients = (array) => {
```

Comment:
nit: The naming of this function is confusing. It reads as if the function returns ingredients that belong to a user.